### ISSUE TITLE
Onboarding color contrast issues when in dark mode

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#15286]
**Describe the bug**

This won't happen when someone onboards the first time, but if they revisit onboarding and have the theme set to dark, they will be unable to read parts of the onboarding.

**To Reproduce**

1. Ensure your settings are configured to have the dark theme, https://dev.to/settings/customization

![image](https://user-images.githubusercontent.com/833231/140531035-06e40796-9b1b-412e-adba-c87850240553.png)

2. Navigate to onboarding, https://dev.to/onboarding
3. Notice that the first step of the onboarding is very difficult to read.

![image](https://user-images.githubusercontent.com/833231/140530350-22d2e5fd-1273-47c4-aeca-8a069aa94716.png)

**Expected behavior**

Regardless of theme, the onboarding should always be readable and have proper colour contras ratios.

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/140530350-22d2e5fd-1273-47c4-aeca-8a069aa94716.png)

![image](https://user-images.githubusercontent.com/833231/140530672-f3cdc013-cae1-442e-9ccc-cc3e554f1de1.png)

This one you can read, but the styles for the cards appear to be off.

![image](https://user-images.githubusercontent.com/833231/140530810-44f17b68-75d3-42ae-a1a3-97594f1b7c97.png)

**Desktop (please complete the following information):**

N/A

**Smartphone (please complete the following information):**

N/A

**Additional context**

**Comment by aitchiss** [comment]
Just thinking this will become crucial to address as we move toward respecting a user's OS settings for light vs dark theme by default

**Comment by nickytonline** [comment]
Good point @aitchiss. cc: @ludwiczakpawel.

**Comment by ludwiczakpawel** [comment]
I'll give it a look!
